packages/flutter/test/flutter_test_config.dart

```diff
@@ -11,6 +11,17 @@ import 'package:leak_tracker/leak_tracker.dart';
 import '_goldens_io.dart'
   if (dart.library.html) '_goldens_web.dart' as flutter_goldens;
 
+/// If true, leak tracking will be enabled for all tests `testWidgetsWithLeakTracking`.
+///
+/// By default, the constant is false.
+/// To enable the leak tracking for all tests, either pass the compilation flag
+/// `--dart-define=flutter_test_config.leak_tracking=true` or
+/// temporarily pass `defaultValue = true` to `fromEnvironment` in the constant definition.
```

Comment:
nit: remove the line starting with 

"or temporarily pass `defaultValue = true` to `fromEnvironment` in the constant definition.
I would think passing in the `--dart-define` should be sufficient for all users.